public/mobile-app/src/lib/api-holidays.ts

```diff
@@ -0,0 +1,70 @@
+import { PUBLIC_API_URL } from '$env/static/public'
+
+export type Holiday = {
+  description: string
+  zones: string
+  start_date: Date
+  end_date: Date
+  emoji: string
+}
+
+export const dayName = (date) =>
+  date.toLocaleDateString('fr-FR', { weekday: 'short' }).replace('.', '')
+
+const capitalizeFirstLetter = (val) =>
+  String(val).charAt(0).toUpperCase() + String(val).slice(1)
```

Comment:
Nit: the following helper `monthName` uses a proper function body with a `return`, not those two. Maybe uniformise those two by using the same form? I'd personnally be happier with the "long" form, with the explicit `return`, what do you think?